File: bug/toto.pl
use Inline (
    Java => 'DATA',
	STUDY => ['java.util.HashMap'],
	AUTOSTUDY => 1,
) ;

my $o = test->f() ;
print $o->[0] ;


__END__
__Java__

class test {
	static public Object f(){
		return new String [] {"allo"} ;
	}
}

File: bug/toto.java
import java.util.* ;

class toto {
	static public void main(String args[]){
		HashMap h = new HashMap() ; 

		h.put("key", "value") ;
		Object valArr[] = h.entrySet().toArray() ;

		for (int i = 0 ; i < valArr.length ; i++){
			System.out.println(valArr[i]) ;
		}
	}
}
